src/components/Editor/Preview/Popup.js

```diff
@@ -96,9 +102,22 @@ export class Popup extends Component {
   }
 
   renderObjectPreview(expression: string, root: Object) {
-    return (
-      <div className="preview-popup">{this.renderObjectInspector(root)}</div>
-    );
+    let content = null;
+    const { loadedObjects } = this.props;
+    const getObjectProperties = id => loadedObjects[id];
+    const roots = this.getChildren(root, getObjectProperties);
```

Comment:
could we filter the roots to just be two properties:

```js
roots = roots.filter(root => ["state", "props"].includes(root.name));

```

Reply:
ya that's already being done in `ReactPopover.js`